applications/search/src/components/filter-option/filter-option.component.jsx

```diff
@@ -36,7 +36,7 @@ export const FilterOption = props => {
     textLabel = localization[optionLabel] || optionLabel;
   }
   if (textLabel && textLabel === textLabel.toUpperCase()) {
-    textLabel = `${_capitalize(textLabel)}`;
+    textLabel = localization[textLabel.toLowerCase()] || `${_capitalize(textLabel)}`;
```

Comment:
Den hele textLabel fallback logikk ser for komplisert ut (L22-L40), vurder hvis kan forenkles

Reply:
Hei! I så fall så trengs det en egen refaktoreringsoppgave på det. Alternativt kan det gjøres neste gang man gjør endringer i filterne.